src/components/Table/table-cells/TableBodyCell.tsx

```diff
@@ -38,7 +37,7 @@ export const TableBodyCell: React.FC<TableCellProps> = ({
     const renderTableCell = () => {
         // These are cells that are in the expandable row (canExpand) and it is the column in
         // which the cells show the chevron icon to expand and collapse sub-rows (isExpandableColumn)
-        if (isExpandableColumn && row?.canExpand) {
+        if (isExpandableColumn && row?.canExpand && row?.depth > 1) {
```

Comment:
@ffffffelix I reverted this change because it was causing a bug in rendering the expansion chevrons. I don't think it was necessary, anyway.